### DISCUSSION TITLE
Github Achievement - The Wanted Feature

### DISCUSSION THREAD

**Discussion by kunal-techprescient** [community/community#53109] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have been using git since last 8 years and it always feels new experience. The achievements section show's couple of badges which user has achieved while working on Github projects. This is one of very good feature and it helps feel motivated and pat on back gesture. I would love to see some more badges in future.

Thank you for making the Github a place we can consider as second home!!! 🚀

**Comment by Merewyn** [comment]
Totally agree! Loved seeing that achievement badge acknowledging progress as features are used! Hope you expand and continue this gamification aspect that provides a fun sense of leveling up.
